### DISCUSSION TITLE
Stack & Heap overflow LED

### DISCUSSION THREAD

**Discussion by anatom74** [feilipu/Arduino_FreeRTOS_Library#36] [Q&A]
How can I change Stack and heap overflow LED blinking (also blink LED on custom port/pin)?

**Comment by Floessie** [comment]
@anatom74 See [`vApplicationMallocFailedHook()`](https://github.com/feilipu/Arduino_FreeRTOS_Library/blob/master/src/variantHooks.cpp#L101) and [`vApplicationStackOverflowHook()`](https://github.com/feilipu/Arduino_FreeRTOS_Library/blob/master/src/variantHooks.cpp#L164).

HTH,
Flössie

**Comment by anatom74** [comment]
got it.

I see that the're weak functions so I managed to implement my own.

Thanks
